alfred-telemetry-platform/src/main/java/eu/xenit/alfred/telemetry/webscripts/console/AdminConsoleWebScript.java

```diff
@@ -48,12 +66,32 @@ public AdminConsoleWebScript(MeterRegistry meterRegistry, Properties globalPrope
         return model;
     }
 
-    private void includeRegistryProperties(Map<String, Object> model, final String registry) {
-        Map<String, String> relevantProps = globalProperties.stringPropertyNames().stream()
-                .filter(s -> s.startsWith(PROP_PREFIX_REGISTRY + registry))
-                .collect(Collectors.toMap(s -> s, globalProperties::getProperty));
+    @VisibleForTesting
```

Comment:
Black magic?

Reply:
Also see: https://jira.sonarsource.com/browse/RSPEC-5803

Reply:
Simply an annotation that this method could have been private, but then I would not be able to access that from the tests.